build-tools/packages/build-cli/src/flags.ts

```diff
@@ -36,14 +40,36 @@ export const packageSelectorFlag = Flags.build({
     description: "package",
     // Can't be used with release groups.
     exclusive: ["g"],
+    multiple: false,
+    parse: async (input) => {
+        const split = input.split("=");
```

Comment:
I think some brief comments explaining the string parsing assumptions here would be good for future maintenance.

Reply:
Will do.

Reply:
I also added a TODO to evaluate whether we want to continue to support this custom parsing.